roles/openshift_efk/tasks/main.yaml

```diff
@@ -0,0 +1,179 @@
+---
+
+  - fail: msg="This role requires the following vars to be defined: master_url, kibana_hostname, es_cluster_size"
+    when: "kibana_hostname is not defined or
+          es_cluster_size is not defined or
+          master_url is not defined"
+
+  - name: "Checking for logging project"
+    command: oc get project logging
+    register: logging_project
+    failed_when: "'FAILED' in logging_project.stderr"
+    tags:
+      - cleanup
```

Comment:
> Also can you explain to me what this line is actually saying: 
> `not openshift_hosted_logging_cleanup | default(false) | bool`

Yes, first it is taking the value of `openshift_hosted_logging_cleanup` and applying the `default` filter, which will set the value to `false` if `openshift_hosted_logging_cleanup` is not defined. Then it is applying the `bool` filter which treats the resulting value of applying the `default` filter to `openshift_hosted_logging_cleanup` as a yaml/ansible boolean value (yes/no, true/false, 1/0, True/False, TRUE/FALSE, etc). Then finally it negates the result.